lib/util.js

```diff
@@ -225,10 +225,15 @@ var util = {
             );
           }
         } else if (currentSection) {
-          var item = line.match(/^\s*(.+?)\s*=\s*(.+?)\s*$/);
-          if (item) {
+          var expression = line.split('=');
+          var isAssignment = expression.length >= 2 && !!expression[0];
+
+          if (isAssignment) {
+            var name = expression[0].trim();
+            var value = expression.slice(1).join('=').trim();
```

Comment:
Use [indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) and [substring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring) instead of split?

```suggestion
          var indexOfEqualsSign = line.indexOf("=");

          if (indexOfEqualsSign !== -1) {
            var name = line.substring(0, indexOfEqualsSign).trim();
            var value = line.substring(indexOfEqualsSign + 1).trim();
```